samples/browser/src/geckoNightly/java/org/mozilla/samples/browser/Components.kt

```diff
@@ -18,6 +18,8 @@ class Components(private val applicationContext: Context) : DefaultComponents(ap
             installWebExtension(WebExtension("mozac-borderify", "resource://android/assets/extensions/borderify/")) {
                 ext, throwable -> Log.log(Log.Priority.ERROR, "SampleBrowser", throwable, "Failed to install ${ext.id}")
             }
+
+            icons.install(this)
```

Comment:
For later: Let's discuss how we want to install extensions. I am currently passing the engine to a `ReaderViewFeature` and install it there. I think all these combinations are fine. Just wondering if we should have a feature-webextensions where all extensions are managed/installed from? otoh, it seems fine for now to have mulitple components just go through the engine.